### DISCUSSION TITLE
Create Plaid payment recipient without address

### DISCUSSION THREAD

**Discussion by yardevx** [TomorrowIdeas/plaid-sdk-php#50] [General]
Hello everyone!
I have a question about Plaid PHP SDK.
I checked Plaid documentation https://plaid.com/docs/api/products/payment-initiation/#payment_initiationrecipientcreate
and it said that address parameter is not required. 
![image](https://user-images.githubusercontent.com/103759817/163583004-e94b8aea-da9c-4164-8ece-e2172053d4be.png)
Can I create Plaid recipient using your SDK without address ?
because until now I found only this method:
`$plaid->payments->createRecipient($name, $IBAN, $address);`
Thank you for your answer.

**Comment by brentscheffler** [comment]
You are correct. I will push out an update that makes Address an optional parameter. Thanks for reaching out and bringing this to my attention.

**Reply by brentscheffler** [reply]
Release 1.0.7 (now available) fixes this issue.
